docs/_includes/common/section-title.html

```diff
@@ -0,0 +1,206 @@
+<div id="section-title">
+    <h2><span class="secondary-text">@ebay/skin/</span>section-title</h2>
+
+    <p>Section titles function as identifiers for groups of elements. They are used in a bold font weight, and should be used when you have a layout that has narrow space, or where the left hand rail is exposed. It's designed to have low impact on vertical space, but still help drive visual hierarchy of the page.</p>
+    
+    <h3>Section title</h3>
+    <p>The standard top heading is designed to support a single line on desktop and wrap only when displayed on narrow screens such as mweb or native.</p>
+
+    <div class="demo">
```

Comment:
We need another `<div class="demo__inner">` child element in each demo div. This extra element helps us identify margins and margin collapse behaviour.

Reply:
@ianmcburnie Is `demo_inner` needed for both documentation and tests?

Reply:
Should be resolved now. I added `demo__inner` wrapper to both docs and tests